### PR TITLE
Snapcast scripts

### PR DESCRIPTION
**Pull request by cz8s**
It would be nice to have a place where contributors could put their little scripts that 
help making snapcast even better.
I will start with this script that helps me automating which stream to play.
This is mostly inspired by #569

### CODE REVIEW

**Comment by xabolcs** [comment]
There is a `/control` directory with nice snippets.

https://github.com/badaix/snapcast/tree/develop/control

**Comment by cz8s** [comment]
Oh, I didn't even see the /control directory. Maybe its just me or is it in a non-obvious place for others too? should I move /control to /contrib/control ?

**Comment by badaix** [comment]
Thanks, I'm still struggeling where to put the scripts

File: contrib/automove/snapswitch.service [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Change all Snapcast groups to play current stream
+After=network.target time-sync.target sound.target avahi-daemon.service
```
Comment (escalade) [inline contrib/automove/snapswitch.service:3]: I think you'll need to add "Wants=" with the same arguments as "After=" for the ordering to work.

File: contrib/automove/snapswitch.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import snapcast.control
+
+loop = asyncio.get_event_loop()
+server = loop.run_until_complete(snapcast.control.create_server(loop, 'localhost'))
+
+def on_stream_update(stream):
+    if stream.status == 'playing':
+	for group in server.groups:
+	    asyncio.run_coroutine_threadsafe(group.set_stream(stream.name), loop)
+	    print("Group %s is now playing stream %s" % (group.friendly_name,stream.name))
```
Comment (djmattyg007) [inline contrib/automove/snapswitch.py:12]: The indentation on these lines is mixed between tabs and spaces.

File: contrib/automove/snapswitch.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+print("Started listening for stream events ....")
+try:
+    for stream in server.streams:
+	stream.set_callback(on_stream_update)
+    loop.run_forever()
```
Comment (djmattyg007) [inline contrib/automove/snapswitch.py:18]: Same here.

**Comment by cz8s** [comment]
I think this PR is not needed anymore since snapcast now supports meta streams.
They work very well for my usecase.

**Pull request closed without merging**